src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -44,6 +45,9 @@ static Command parse(String input) throws WrongFormatException,
                 ListCompanyCommand companyCommand = new ListCompanyCommand(command + " companies");
                 return companyCommand;
             } else if (inputWords[1].equals("venues")) {
+                if (inputWords.length > 2){
```

Comment:
Perhaps put a space in between the closing parenthesis and the opening curly braces